### DISCUSSION TITLE
Explorers de Colombia 🇨🇴

### DISCUSSION THREAD

**Discussion by NiiSV811** [Launch-X-Latam/MisionFrontEnd#103] [General]
Presentense Explorers de Colombia. Comenten de donde son, a que se dedican, etc... A y como encontraron LaunchX

**Comment by jlnunezobando** [comment]
Hola Cali, Ingeniero Electrónico, Youtube

**Comment by CGux8** [comment]
si buenas, soy de palmira, tecnologo en sistemas como del 2011 pero apenas retomando la programacion xD

**Comment by TORVIICT** [comment]
Holis, soy de Cali, tecnico de sistemas, google news.
